qemu/tests/block_hotplug.py

```diff
@@ -90,6 +93,7 @@ def get_new_disk(disk1, disk2):
                 disks_before_plug = find_disk(vm, get_disk_cmd)
 
                 if params.get("need_controller", "no") == "yes":
+                    remove_controller = True
```

Comment:
just want to check with you, does `need_controller` mean the guest needs hot-plug controller ?

Reply:
hachen, yes